sba-business-rules.md

```diff
@@ -0,0 +1,37 @@
+---
+title: DATA Act business rules
+---
+As an SBA user, I need to see the business rules being applied to my source data
+
+Business rules or validation rules will be applied to ensure that submitted data by the US Small Business Administration meets the DATA Act criteria. Goals of these validation rules will improve cleanliness, accuracy, and meaningfulness of the data set. Below is a draft of proposed rules and are subject to change. 
+
+*Standard validation*
+
+Required fields: 
+	Required field validation ensures that users input data in required fields. If field is empty, an error will be raised. Required fields to be determined in the [DATA act schema](https://github.com/18F/intercessor/blob/master/schema/data-act-schema.png)
```

Comment:
* Just to be as specific as possible: "If a _required_ field is empty, and error will be raised." 
* since the DATA Act schema doesn't really determine the required fields (it just reflects them), might be more accurate to say that "the DATA Act schema documents the required fields"